src/theory/quantifiers/cegqi/ceg_arith_instantiator.cpp

```diff
@@ -191,7 +191,7 @@ bool ArithInstantiator::processAssertion(CegInstantiator* ci,
         }
         else
         {
-          Assert(d_type.isReal());
+          Assert(d_type.isArithmetic());
```

Comment:
```suggestion
          Assert(d_type.isReal());
```
The integer case is handled above.